CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.17.1 (08/25/2021)
+
+* [chore]: Replaced the use of deprecated functions in Guzzlehttp/Psr7 and bumbed the version from **^1.2** to **^1.7|^2.0**
+
```

Comment:
Please remove this change, as this will be added in a separate release PR. Also this is more of a `feat` than a bug.